FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1/c_sdk/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -619,7 +619,8 @@ bool _IotMqtt_ValidateConnect( const IotMqttConnectInfo_t * pConnectInfo )
         /* If will info is provided, check that it is valid. */
         if( pConnectInfo->pWillInfo != NULL )
         {
-            status = _IotMqtt_ValidateLwtPublish( pConnectInfo->awsIotMqttMode,
+            status = ( pConnectInfo->pWillInfo->pPayload != NULL ) &&
```

Comment:
I find this coding style bit odd. But since it is ported from v4_beta, I'm approving it.